partcad/src/partcad/shape.py

```
@@ -562,326 +561,322 @@ def error(self, msg: str):
-                try:
-                    result = shape_envelope.deserialize(cleaned_response)
-                except Exception as e:
-                    pc_logging.error(f"Failed to deserialize response: {e}")
-
-                if not result.get("success", False):
-                    pc_logging.error(
-                        f"Render {format_name.upper()} failed for {self.project_name}:{self.name}: {result.get('exception', 'Unknown error')}"
-                    )
-                if "exception" in result and result["exception"]:
-                    pc_logging.exception(f"Render {format_name.upper()} exception: {result['exception']}")
+            for format in [format_name] if format_name else output.all_formats(ctx):
```

Ruff (0.16.1)

[error] Variable `format` is shadowing a Python builtin

(A001)